easybuild/easyconfigs/p/picard/picard-2.18.11-Java-1.8.0_162.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Insitute, London, UK
+##
```

Comment:
@jdemeul Please retain the credit for work on previous versions in the comments.